homeassistant/components/samsungtv/config_flow.py

```diff
@@ -283,33 +311,45 @@ async def async_step_zeroconf(self, discovery_info: DiscoveryInfoType):
         self.context["title_placeholders"] = {"device": self._title}
         return await self.async_step_confirm()
 
-    async def async_step_confirm(self, user_input=None):
+    async def async_step_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> data_entry_flow.FlowResult:
         """Handle user-confirmation of discovered node."""
         if user_input is not None:
 
             await self.hass.async_add_executor_job(self._try_connect)
+            if self._bridge is None:
+                raise ConnectionError("Unable to connect to %s" % self._host)
             return self._get_entry_from_bridge()
 
         self._set_confirm_only()
         return self.async_show_form(
             step_id="confirm", description_placeholders={"device": self._title}
         )
 
-    async def async_step_reauth(self, data):
+    async def async_step_reauth(
+        self, data: MappingProxyType[str, Any]
+    ) -> data_entry_flow.FlowResult:
         """Handle configuration by re-auth."""
         self._reauth_entry = self.hass.config_entries.async_get_entry(
             self.context["entry_id"]
         )
+        if not self._reauth_entry:
+            return self.async_abort(reason="missing_config_entry")
```

Comment:
"missing_config_entry" is not in the translations

Reply:
Added